pkg/api/mlflow/dao/repositories/run.go

```diff
@@ -237,3 +250,37 @@ func (r RunRepository) SetRunTagsBatch(ctx context.Context, run *models.Run, bat
 	}
 	return nil
 }
+
+// getMinRowNum will find the lowest row_num for the slice of runs
+// or 0 for an empty slice
+func getMinRowNum(runs []models.Run) models.RowNum {
+	// get the lowest row_num in the slice
+	minRowNum := models.RowNum(0)
+	for _, run := range runs {
+		if minRowNum == models.RowNum(0) || run.RowNum < minRowNum {
```

Comment:
my suggestion to compare to the zero value was to avoid setting `minRowNum` to an invalid value, but it's not useful anymore now that we're checking that there are no non-existent run IDs provided

Reply:
i think we need this to get the first value, otherwise the zero-value will always be lowest

Reply:
ah yes, good point!